Tests/ViewInspectorTests/SwiftUI/TreeViewTests.swift

```diff
@@ -28,6 +28,16 @@ final class TreeViewTests: XCTestCase {
         }
         XCTAssertEqual(sut.content.medium.viewModifiers.count, count)
     }
+
+  func testVariadicViewTree() throws {
+    let button = try VariadicViewScreen().inspect().find(button: "Change opacity")
```

Comment:
It could be interesting to verify that a child view has the expected opacity applied.

Reply:
The blocking issue I see is that the opacity is applied by the `root` of the `Tree` - `VariadicViewScreenOpacityRoot`. And the test code does not call into it. It only calls the `content` property.

Reply:
Based on [this](https://movingparts.io/variadic-views-in-swiftui) article @rafael-assis I thought that the opacity would be applied to each subview when the root is `_Variadic­View_Multi­View­Root` like we have here. However, I think an issue is that there's no way to read the opacity from the subviews.

Reply:
It's correct that the `_Variadic­View_Multi­View­Root` will apply the opacity to each view. But I believe that this will happen in the context of the SwiftUI layout pass. In the context of ViewInspector test the `VariadicViewScreen`'s `body` property is called but the `func body(children: _VariadicView.Children) -> some View` of the `VariadicViewScreenOpacityRoot` is never called.

Reply:
I can have a closer look myself at some point, but I have a feeling we might need to explicitly support `_Variadic­View.Multi­View­Root` by defining a `ViewType`, like for `ForEach`, and similarly mark it as `MultipleViewContent`. Then we'll have full control on children extraction, including ability to maintain `opacity` modifiers. Usually modifiers are reset like so: `content.medium.resettingViewModifiers()`

As a side note, you can use `Inspector.print(view) as AnyObject` instead of `dump` when printing view's internals, it's doing a bit extra work when digging into the structure.

Reply:
It makes a lot of sense.
This seems to be beyond the scope of what we need right now as we're only looking to find the views that are contained in a Layout based view.
But it would definitely be great to have full support for variadic views in a way that the root of the tree is called to apply all the modifiers to the content views.